src/Runner.Worker/action_yaml.json

```diff
@@ -120,6 +128,24 @@
             ],
             "string": {}
         },
+        "composite-output-value": {
+            "context": [
+                "github",
+                "needs",
+                "strategy",
+                "matrix",
+                "secrets",
+                "steps",
+                "inputs",
+                "job",
+                "runner",
+                "env"
+            ],
+            "one-of": [
+                "string",
+                "mapping"
```

Comment:
we add mapping as well as to support something like this which follows the same format as container's outputs:
```
outputs:
 time: # id of output
   description: 'The time we did the greeting'
```